inference/core/interfaces/http/http_api.py

```diff
@@ -956,6 +960,41 @@ async def infer_lmm(
 
         if not DISABLE_WORKFLOW_ENDPOINTS:
 
+            @app.post(
```

Comment:
that needs to be added into hosted API routing

Reply:
I think we need to add it here like the other ones:  https://github.com/roboflow/roboflow-infra/blob/1e58b097dc081a18851588a253d18ac3e4539b1c/aws/serverless/modules/containerlambda/main.tf#L128-L144

Reply:
@EmilyGavrilenko is the customer requesting this running on hosted API or dedicates/local?

Reply:
that's hosted most likely given Nick comment
I posted this comment not to forget

Reply:
regardless - this is potentially useful feature for preview in UI

Reply:
infra PR here:  https://github.com/roboflow/roboflow-infra/pull/334